bin/install.sh

```diff
@@ -341,6 +346,20 @@ echo WEB_CONCURRENCY=$webConcurrency >> $envFile
 
 echo -e "\n\nGreat! That's all we need, we will have your Waffle.io Takeout ready shortly."
 
+######################################################
+# Store off environment configuration in /etc/waffle #
+######################################################
+echo -e "\n\nSaving environment configuration in /etc/waffle/"
+sudo mkdir -p /etc/waffle/ca-certificates
+sudo cp $envFile /etc/waffle/environment.list
```

Comment:
I'm not sure what the best approach is yet. We need to account for both a clean install and updating an existing install. Namely, what if the user has set some environment variables already, but we need to add some, change some, and/or default some. Not super obvious what the right answer is.